packages/api/src/platforms/vtex/resolvers/setPassword.ts

```diff
@@ -0,0 +1,26 @@
+import type { Context } from '..'
+import type {
+  MutationSetPasswordArgs,
+  SetPasswordResponse,
+} from '../../../__generated__/schema'
+import { BadRequestError } from '../../errors'
+
+export const setPassword = async (
+  _: any,
+  { data }: MutationSetPasswordArgs,
+  { clients: { commerce } }: Context
+): Promise<SetPasswordResponse> => {
+  if (!data?.email) {
+    throw new BadRequestError('Missing email')
+  }
+
+  const response = await commerce.setPassword({
```

Comment:
If you agree with the previous suggestion, you also need to change here.
```suggestion
  const response = await commerce.vtexid.setPassword({
```